treeherder/model/models.py

```diff
@@ -18,9 +18,11 @@
 from django.forms import model_to_dict
 from django.utils import timezone
 from django.utils.encoding import python_2_unicode_compatible
+from more_itertools import flatten
```

Comment:
The `flatten` method is pretty simple:
https://github.com/erikrose/more-itertools/blob/96c4136dd4cb47a18f2522dff6bb113c9bf36860/more_itertools/recipes.py#L236-L245
(and actually comes from https://docs.python.org/2/library/itertools.html#recipes)

I'm not 100% against adding a dependency on [more_itertools](https://github.com/erikrose/more-itertools), however curious to hear your thoughts for and against?

Reply:
I always forget how simple `flatten` is!

re the recipe: more-itertools packages all the recipes from there with some extras thrown in, it's really useful!

re adding it as a dep: given how simple flatten is, I'm not fussed about including the whole package.  On the one hand it's very stable and I regularly find myself using a few methods from it, on the other it's another dep with upgrade overhead.

Reply:
Perhaps we can save adding the dep until a time where we use a second method from the package?

Reply:
Updated and rebased!